Add fingerprinting-protection-debugger.xpi

FKLC opened the pull request:
Hello,

I'm following [Adding a new xpi](https://github.com/mozilla-extensions/xpi-manifest/blob/main/docs/adding-a-new-xpi.md) steps for [mozilla-extensions/fingerprinting-protection-debugger](https://github.com/mozilla-extensions/fingerprinting-protection-debugger), and this should be the required step for [Enabling releases](https://github.com/mozilla-extensions/xpi-manifest/blob/main/docs/adding-a-new-xpi.md#enabling-releases), but if anything is missing, please do let me know, I can update accordingly.

Thank you!

gabrielBusta commented on taskcluster/config.yml:
```diff
@@ -123,6 +123,12 @@ taskgraph:
             default-repository: https://github.com/mozilla-extensions/privileged-test-xpi
             default-ref: main
             type: git
+        fingerprinting-protection-debugger:
```
```suggestion
        fingerprintingprotectiondebugger:
```

gabrielBusta commented:
Could you provide more context around this addon? Can I take a look at the request to create this repo? (in Bugzilla maybe?)

gabrielBusta commented:
Also, [@example.com](https://github.com/mozilla-extensions/fingerprinting-protection-debugger/blob/08816ec62d914ab62c724d676e79a4fb090a4116/src/manifest.json#L11) is not a valid suffix for the addon id.

FKLC commented:
> Could you provide more context around this addon? Can I take a look at the request to create this repo? (in Bugzilla maybe?)

I believe we don't have a bug on bugzilla for this extension, but we had some conversation on [#addonds-pipeline here](https://mozilla.slack.com/archives/CMKP7NPKN/p1715361932108379) (it also includes the request to create the repo). To give some context, this is a very simple extension that let's an advanced user to debug RFP targets (resist fingerprinting protection targets). Some RFP targets break websites, but it is sometimes hard to pinpoint which [RFPTarget](https://searchfox.org/mozilla-central/source/toolkit/components/resistfingerprinting/RFPTargets.inc) is responsible for breakage. AFAIK, another use for this extension is going to be, we'll store and apply RFP target overrides per-site, so that the advanced user can browse websites without modifying preferences all the time.

> Also, [@example.com](https://github.com/mozilla-extensions/fingerprinting-protection-debugger/blob/08816ec62d914ab62c724d676e79a4fb090a4116/src/manifest.json#L11) is not a valid suffix for the addon id.

Yes, sorry about that. I fixed it in dev branch. I moved to dev branch assuming it would become the production branch, and gets restricted.

willdurand commented on manifests/fingerprinting-protection-debugger.yml:
```diff
@@ -0,0 +1,13 @@
+repo-prefix: fingerprinting-protection-debugger
+active: true
+private-repo: false
+additional-emails: ["fkilic@mozilla.com"]
```
Please add a group email if possible, otherwise more individuals should probably be listed.

FKLC replied:
I just added Frederik Braun, Tom Ritter, and Tim Huang to the list. Thank you!

willdurand commented on manifests/fingerprinting-protection-debugger.yml:
```diff
@@ -0,0 +1,13 @@
+---
+description: Fingerprinting Protection Debugger Addon
+repo-prefix: fingerprinting-protection-debugger
```
should have been:

```suggestion
repo-prefix: fingerprintingprotectiondebugger
```
